site/content/zh-hans/docs/getting-started/introduction.md

```diff
@@ -6,7 +6,7 @@ title: 概览简介
 
 HTNN（Hyper Trust-Native Network）是一款基于云原生技术的 L3&L4&L7 Cross-Layer 网络全局解决方案产品。
 
-![HTNN](/images/HTNN.png)
+![HTNN](/site/static/images/HTNN.png)
```

Comment:
When the website is built, the images will be copied into /public/images, so we have to use /images/HTNN.png, not the link used on the GitHub.